mailbox/store/src/main/java/org/apache/james/mailbox/store/mail/model/FlagsFactory.java

```diff
@@ -114,6 +115,7 @@ public Flags build() {
                         toUserFlagStream(flagsOrEmpty),
                         userFlags.stream())
                     .distinct()
+                    .filter(s -> !Strings.isNullOrEmpty(s))
```

Comment:
requires a test case or it doesn't fix anything

Reply:
DONE